### DISCUSSION TITLE
I get "Docker invalid reference format." when trying to install/use CLI

### DISCUSSION THREAD

**Discussion by Pouletgrill** [immich-app/immich#8101] [Q&A]
Hello, I need to upload a bunch of photos from an old immich library and someone told me i need to use CLI to do that by script. 
I try to run this command in powershell as mentionned in the documentation, but i get a "Docker invalid reference format." :
`docker run -it -v "$(pwd)":/import:ro -e IMMICH_INSTANCE_URL=http://192.168.3.50:2283/api -e IMMICH_API_KEY=API_KEY ghcr.io/immich-app/immich-cli:latest`
keep in mind that yes i did generate an API Key and put it in the command line. 

Can you help me ? 
Thank you.

**Comment by schuhbacca** [comment]
Maybe? https://stackoverflow.com/questions/45682010/docker-invalid-reference-format

**Comment by Pouletgrill** [comment]
I manage to make it work with Node.JS but never with docker. Maybe the documentation need to be revised
